src/lib.rs

```diff
@@ -31,7 +31,10 @@
 //! [this one]: https://funduinoshop.com/elektronische-module/displays/lcd/16x02-i2c-lcd-modul-hintergrundbeleuchtung-blau
 //! [lcd address]: https://www.ardumotive.com/i2clcden.html
 
+
```

Comment:
I think it's a good idea to add a `#[cfg(not(feature = "async"))]` here.

Reply:
Why? I don't see an issue with having both namespaces available so users can choose for themselves. 
For example, when testing I often switched between sync and async by just changing the import. Since the async I2C implementation also implements the blocking methods I didn't have to change anything else. It would have been more work to change the features everytime

Reply:
I don't have a hard opinion on that :)